docs/scheduled_tasks.md

```diff
@@ -0,0 +1,25 @@
+# Scheduled tasks
+
+You need to configure scheduled tasks in order to properly run your marketplace in production.
+
+Scheduled tasks are tasks that the system needs to run every now and then to keep the application working as expected. Operations such as sending mass emails to all users or cleaning up old data from the database are done in scheduled tasks.
+
+## List of required tasks
+
+Here's a list of scheduled tasks you need to configure:
```

Comment:
In technical documentation and formal writing contractions should be avoided. So, `Here is` instead of `Here's`.